source/auth.js

```diff
@@ -4,10 +4,10 @@ function generateBasicAuthHeader(username, password) {
     return "Basic " + Buffer.from(username + ":" + password).toString("base64");
 }
 function generateTokenAuthHeader(tokenInfo) {
-    return `${tokenInfo.token_type} ${tokenInfo.access_token}`;
+    return tokenInfo.token_type + " " + tokenInfo.access_token;
```

Comment:
No need for this - Babel is used before publishing, so this will be transpiled. The tests pass for node 4, which is my earliest support version:

![image](https://user-images.githubusercontent.com/3869469/37846476-61083c7a-2ed6-11e8-88de-2ee62603f7ea.png)

Anything outside of this is not currently in scope.